src/components/Menu/CollapseOptions.tsx

```diff
@@ -0,0 +1,44 @@
+import { css } from '@emotion/css';
+import ArrowBottom from '@icon/ArrowBottom';
+import React, { memo } from 'react';
+
+const style = css`
+  height: 12px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  & > div {
+  }
+
+  .border-b {
+    width: 100%;
+    flex: 1;
+  }
+
+  .x5 {
+    padding: 0 5px;
+    border-radius: 3px;
+    cursor: pointer;
+  }
+  
+  .fold {
+    transform: rotate(180deg);
+  }
+`;
```

Comment:
To ensure consistency throughout the project, please use tailwindcss

Reply:
```txs
<div className='flex items-center'>
      <div className='border-b border-white/20 flex-1'></div>
      <div className='bg-gray-900 hover:bg-gray-500/10 transition-colors duration-200 px-3 rounded-sm cursor-pointer'
           style={ { transform: props.fold ? 'rotate(180deg)' : '' } }
           onClick={ () => props.onSetFold(!props.fold) }>
        <ArrowBottom />
      </div>
      <div className='border-b border-white/20 flex-1'></div>
    </div>
```